week-3/D-array-filter/exercise.js

```diff
@@ -8,7 +8,12 @@
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-var pairsByIndex; // Complete this statement
+var pairsByIndex = pairsByIndexRaw.filter(arrayElemnet => {
+  if (arrayElemnet !== null)
+    return (
+      typeof arrayElemnet[0] === "number" && typeof arrayElemnet[1] === "number"
```

Comment:
This is correct but we could make this shorter by *chaining* the conditionals together. Something like this:

```js
return myVariable !== null && typeof myVariable[0] === "number"
```